src/CertificateChainValidationEngine.js

```diff
@@ -1046,10 +1046,10 @@ export default class CertificateChainValidationEngine
 			/// <returns type="Boolean">Boolean result - valid or invalid the "name" against the "constraint"</returns>
 			
 			//region Initial check
-			if((name.typesAndValues.length === 0) || (constraint.typesAndValues.length === 0))
+			if((name.relativeDistinguishedNames.length === 0) || (constraint.relativeDistinguishedNames.length === 0))

```

Comment:
This PR request changes API. It requires updating all PKIjs dependents.

@rviau42 Is it possible to stay  `name.typesAndValues` field for v2?

Maybe just add `typesAndValues` getter/setter as a mirror of `relativeDistinguishedNames` and mark it as a deprecated field. Is it possible?

@rmhrisk What do you think about these API changes?

Reply:
@microshine I thought about that but I rejected this idea in first place (due to certificate structure changes).
It could work only if the `typesAndValues` getter/setter is the mirror of the first element of `relativeDistinguishedNames`.
But there could be some side-effects.

Reply:
@rviau42 Could you add getter/setter and constructor parameters for`typeAndValues` to `DistinguishedName` class?

We need to save our current API without changes

Something like

```js
/**
 * @deprecated
 */
get typeAndValues() {
  return this.relativeDistinguishedNames[0];
}

/**
 * @deprecated
 */
set typeAndValues(v) {
  this.relativeDistinguishedNames[0] = v;
}
```